cpp-throughput/throughput_scaling.cpp

```diff
@@ -114,9 +134,15 @@ int main(int argc, char* argv[]) {
     std::string s_bytes(argv[1]);
     int n_bytes = std::stoi(s_bytes);
 
+    log_data(context, LLDebug, "Starting Throughput tests for rank: " + std::to_string(rank));
     double main_start = MPI_Wtime();
 
+    //Indicating tensor size used in throughput test
+    std::string tensor_text = "Running throughput scaling test with tensor size of ";
```

Comment:
We'll know from the previous log message that we're in the throughput scaling test. All we really need here is the tensor size, right?  Better not to be too verbose in log messages.

Reply:
Hmmm I see where it throws an error if the number of tensor bytes is not added? But I do not see it outputted twice, I might also be missunderstanding